cdap-common/bin/functions.sh

```diff
@@ -207,13 +207,7 @@ cdap_stop_pidfile() {
 #
 cdap_check_pidfile() {
   local readonly __pidfile=${1} __label=${2:-Process}
-  if [[ -f ${__pidfile} ]]; then
-    local readonly __pid=$(<${__pidfile})
-    if [[ $(cdap_status_pidfile ${__pidfile} ${__label} >/dev/null) ]]; then
-      echo "${__label} running as PID ${__pid}. Stop it first."
```

Comment:
Ehh, status most definitely should not say "stop it first" anywhere. We are discussing the behavior when a start is attempted while we're already running, which is what's handled here.